### ISSUE TITLE
how to pass a param to VideoSnapshot when live stream

### ISSUE THREAD

**Issue by dragonishare** [zzarcon/video-snapshot#10]
how to pass a param to VideoSnapshot when live stream
when live stream, new VideoSnapshot(), how can i pass its param

**Comment by mgw-sbex** [comment]
Yes we would like this too, or as an alternative, can we just pass in a player instead of a file?
